sdk/documentintelligence/Azure.AI.DocumentIntelligence/README.md

```diff
@@ -169,41 +168,28 @@ We guarantee that all client instance methods are thread-safe and independent of
 
 The following section provides several code snippets illustrating common patterns used in the Document Intelligence .NET API. Most of the snippets below make use of asynchronous service calls, but keep in mind that the Azure.AI.DocumentIntelligence package supports both synchronous and asynchronous APIs.
 
-* [Use the Prebuilt General Document Model](#use-the-prebuilt-general-document-model)
+* [Extract Layout](#extract-layout)
```

Comment:
Since we'll be removing the general document model sample, decided to include the layout extraction one instead. They're very similar.